lib/filter/enemy.js

```diff
@@ -99,7 +99,31 @@ exports.update = function (json) {
       }
     }
     */
-
+   
+		console.log( 'Enemy Drops:');
+		var drops = 0;
+		for (enemy in json[round].enemy) {
+			if (json[round].enemy[enemy].children[part].drop_item_list.length > 0) {
```

Comment:
I think, there is no reason to duplicate the for loop for the enemies, we already have one above and since you are using here the variable `part`, which is only defined in the nested for loop above, you should move your code below line 81